js/main.js

```diff
@@ -38,32 +64,140 @@ function renderPic(data) {
   $saveButton.setAttribute('class', 'picButton');
   $saveButton.setAttribute('type', 'button');
   $saveButton.textContent = 'SAVE';
+  $saveButton.setAttribute('data-pic', data.message);
+
+  // $allEntries.appendChild($onePic);
+  $onePic.appendChild($picHolder);
+  $onePic.appendChild($buttonHolder);
+  $picHolder.appendChild($picView);
+  $buttonHolder.appendChild($saveButton);
+
+  return $onePic;
+
+}
+
+// for  2 pic showing
+/*
+    <div class=" morePic column-half" data-pic="">
+      <div class="picHolder"><img class="picView-two" src="https://images.dog.ceo/breeds/hound-walker/n02089867_3103.jpg"></div>
+      <div class="saveButtonHolder">
+       <button class="picButton" type="button" data-pic="">SAVE</button>
+      </div>
+    </div>
+   */
+
+function renderMorePic(data) {
```

Comment:
Shouldn't this be named `renderTwoPics`? Also, I would suggest trying to make one function to render the results for any amount of pictures. You might need a conditional for the length of the array of `data` to determine classes to apply, but beyond that, the process seems very similar.